packages/context/test/unit/resolver.test.ts

```diff
@@ -320,3 +326,90 @@ function customAsyncDecorator(def: Object) {
     return f + ':' + b;
   });
 }
+
+describe('method injection', () => {
+  let ctx: Context;
+
+  before(function() {
+    ctx = new Context();
+    ctx.bind('foo').to('FOO');
+    ctx.bind('bar').to('BAR');
+  });
+
+  it('can resolve method arguments', () => {
```

Comment:
Naming convention: we don't start test names with "can" or "should" - see http://loopback.io/doc/en/contrib/style-guide.html#test-naming

```
it('resolves method arguments')
```

Reply:
(This comment applies to all other test names in this pull request too.)

Reply:
+1